tasks/simplifications.json

```diff
@@ -355,6 +355,20 @@
 		"maxmemory": "8192",
 		"skipfailures": "no"
 	},
+	"uk.odl.importpostcodedistricts_": {
```

Comment:
Just out of curiosity, what does the final `_` mean?

Reply:
It has no special meaning. That's just the `table_id` that we get from the task id using this method: https://github.com/CartoDB/bigmetadata/blob/master/tasks/base_tasks.py#L536-L543